src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -1524,181 +1565,178 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
-                                                "\", invalid calculation method=" + cAlphaArgs(4));
+                                            format("{}{}=\"{}\", invalid ACH (air changes per hour) specification [<0.0]={:.3R}",
+                                                   RoutineName,
+                                                   cCurrentModuleObject,
+                                                   thisVentilation.Name,
+                                                   rNumericArgs(5)));
                             ErrorsFound = true;
                         }
                     }
+                    if (lAlphaFieldBlanks(4)) {
+                        ShowWarningError(state,
+                                         std::string{RoutineName} + cCurrentModuleObject + "=\"" + thisVentilation.Name + "\", " +
+                                             cAlphaFieldNames(4) + " specifies " + cNumericFieldNames(4) +
+                                             ", but that field is blank.  0 Ventilation will result.");
+                    }
+                    break;
+
+                default:
+                    if (Item1 == 1) {
+                        ShowSevereError(state,
+                                        std::string{RoutineName} + cCurrentModuleObject + "=\"" + cAlphaArgs(1) +
+                                            "\", invalid calculation method=" + cAlphaArgs(4));
+                        ErrorsFound = true;
+                    }
                 }
 
-                {
-                    auto const SELECT_CASE_var(cAlphaArgs(5)); // Fan type character input-->convert to integer
-                    if (SELECT_CASE_var == "EXHAUST") {
-                        state.dataHeatBal->Ventilation(Loop).FanType = DataHeatBalance::VentilationType::Exhaust;
-                    } else if (SELECT_CASE_var == "INTAKE") {
-                        state.dataHeatBal->Ventilation(Loop).FanType = DataHeatBalance::VentilationType::Intake;
-                    } else if ((SELECT_CASE_var == "NATURAL") || (SELECT_CASE_var == "NONE") || (SELECT_CASE_var == std::string())) {
-                        state.dataHeatBal->Ventilation(Loop).FanType = DataHeatBalance::VentilationType::Natural;
-                    } else if (SELECT_CASE_var == "BALANCED") {
-                        state.dataHeatBal->Ventilation(Loop).FanType = DataHeatBalance::VentilationType::Balanced;
-                    } else {
-                        if (Item1 == 1) {
-                            ShowSevereError(state,
-                                            std::string{RoutineName} + cCurrentModuleObject + "=\"" + state.dataHeatBal->Ventilation(Loop).Name +
-                                                "\". invalid " + cAlphaFieldNames(5) + "=\"" + cAlphaArgs(5) + "\".");
-                            ErrorsFound = true;
-                        }
+                if (cAlphaArgs(5).empty()) {
```

Comment:
What is our new policy for using `cAlphaArgBlanks(5)` vs. `cAlphaArgs(5).empty()` or ... does it not matter because of epJSON?

Reply:
That would be a great change I think.  As long as the DataIPShortcuts::cAlphaArgs array gets reset to empty strings when called.  In the before time, with Fortran, they would be fixed length, so the blanks were a shortcut to having to first trim then check the length of the input value.  I'm not sure the worth of that right now though, given the input processor overhaul coming.